### DISCUSSION TITLE
reclassifying non-wear

### DISCUSSION THREAD

**Discussion by Suesmith12** [Ergo-Tools/ActiPASS#58] [Help and Q&A (processing, quality checking or statistics)]
Is it possible to reclassify behaviours at all? For instance, where a participant was lying for a period and ActiPASS classified this as non-wear (0.3hr), I would like to reclassify this back into wear data? 

I can see in the Diary instructions about classifying MNW events, but can’t see if it is possible to do this the reverse way.

**Comment by pasan-h** [comment]
> Is it possible to reclassify behaviours at all? For instance, where a participant was lying for a period and ActiPASS classified this as non-wear (0.3hr), I would like to reclassify this back into wear data?

Unfortunately this is not possible. ActiPASS non-wear algorithm can sometimes misclassify wear time as non-wear (specially during an undetected bedtimes. During bedtimes, non-wear algorithm sensitivity is lower). However it's also worth considering whether the detected non-wear is real. Any chance of participant removing the sensor for a short period?

> I can see in the Diary instructions about classifying MNW events, but can’t see if it is possible to do this the reverse way.

It's not possible. However we can try to manually define a bedtime (a "Night" or "Bed" event) in the diary (assuming the issue is due to undetected bedtime putting the algorithm into high sensitivity mode) for this person and see what happens. 

If a quality-check figure for this day is attached perhaps we can see whether there are other warnings/issues which can explain this unwanted non-wear episode.
